components/brave_wallet/browser/filecoin_keyring.cc

```diff
@@ -88,6 +89,54 @@ bool FilecoinKeyring::DecodeImportPayload(
   return true;
 }
 
+// static
+bool FilecoinKeyring::GetProtocolFromAddress(
```

Comment:
Didn't notice this earlier, this should be using absl::optional rather than out parameter, but I'm fine if you want to fix it in the next follow-up PR to avoid re-run CI on this one.